library/src/main/java/com/wultra/android/mtokensdk/oidc/models/OIDCConfig.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2025 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.wultra.android.mtokensdk.oidc.models
+
+data class OIDCConfig(
+    /**
+     * Provider's identifier.
+     */
+    val providerId: String,
+
```

Comment:
You have empty lines between properties here, but not in other files - we should be consistent.

Also, consider doing the comments in a "oneliner" way `/** comment here */` to keep it more compact (also aplies for other files). But its just for consideration.